vita3k/modules/SceCommonDialog/SceCommonDialog.cpp

```diff
@@ -1002,8 +1004,10 @@ EXPORT(int, sceSaveDataDialogGetResult, Ptr<SceSaveDataDialogResult> result) {
     result.get(host.mem)->result = host.common_dialog.result;
     result.get(host.mem)->buttonId = host.common_dialog.savedata.button_id;
     result.get(host.mem)->slotId = host.common_dialog.savedata.slot_id[host.common_dialog.savedata.selected_save];
-    result.get(host.mem)->slotInfo.get(host.mem)->isExist = host.common_dialog.savedata.slot_info[host.common_dialog.savedata.selected_save].isExist;
-    result.get(host.mem)->slotInfo.get(host.mem)->slotParam = host.common_dialog.savedata.slot_info[host.common_dialog.savedata.selected_save].slotParam;
+    if (result.get(host.mem)->slotInfo) {
```

Comment:
make result.get(host.name)->slotparam.get(host.name) and host.commondialog.savedialog temp vars